consensus/spos/bls/sovereignSubRoundEnd.go

```diff
@@ -184,9 +186,19 @@ func (sr *sovereignSubRoundEnd) sendOutGoingOperations(ctx context.Context, data
 		return
 	}
 
+	sr.resetOutGoingOpTimer(data)
```

Comment:
You reset the timer after sending the operations only for the leader. We need to reset the timer for all validators on end round.

Reply:
It is explained in the PR description + TODO + task why this isn't implemented